pootle_pytest/fixtures/site.py

```diff
@@ -20,9 +20,16 @@ def site_matrix(request, system, settings):
         TranslationProjectFactory, StoreFactory, UnitFactory)
 
     from pootle_store.models import UNTRANSLATED, TRANSLATED, FUZZY, OBSOLETE
+    from pootle_app.models import Directory
 
-    # create root and projects directories
-    DirectoryFactory(name="projects", parent=DirectoryFactory(parent=None))
+    # create root and projects directories, first clear the class cache
```

Comment:
because of the questionable caching on Directory.objects.root and Directory.objects.projects